+brillem/readparam.m

```diff
@@ -107,7 +111,7 @@
         inputValid = true;
         
         % Go through all dimension of the selected field to check size.
-        if check
+        if 0 % check
```

Comment:
Was this a temporary change or should this size check be removed entirely?

Reply:
I didn't know what `check` was (it's not defined in the file) and I didn't want to remove the block. There are other parts of the code which I've just commented out and not removed elsewhere.

Reply:
This file is a slightly modified version of [sw_readparam.m](https://github.com/SpinW/spinw/blob/master/swfiles/sw_readparam.m) from `spinW` and I mistakenly cut-out too much in the transfer.
Verifying that the input `format` `struct` has a field `'sizes'` is _probably_ good enough for use inside of `brillem`
```suggestion
        if isfield(format, 'sizes')
```